aces/Loris
# [candidate_parameters] Refresh on update

## Brief summary of changes

- Update candidate parameter pages to refresh on save
- This fixes an issue CCNA was facing where changes made in one tab were not visible in another tab unless the user refreshed the page. This was confusing to users so it is better when the tab refreshes automatically.

#### Testing instructions (if applicable)

1. Open Candidate Parameters
2. Try changing fields in different tabs and confirm that after successful input, a confirmation popup appears. After pressing OK, the page should reload


[CCNA OVERRIDE PR 
](https://github.com/aces/CCNA/pull/7135)
